data/anaconda_options.txt

```diff
@@ -269,3 +269,9 @@ timeout in seconds specified as a mandatory value of the option.
 decorated
 Run GUI installer in a decorated window. By default, the window is not decorated, so it doesn't have a title bar,
 resize controls, etc.
+
+blscfg
+Configure GRUB to use the "BootLoader Spec" boot option configuration format.
+
+noblscfg
+Configure GRUB not to use the "BootLoader Spec" boot option configuration format.
```

Comment:
Why do we need to have both as boot options? Isn't one of them default behavior?

Reply:
Well, you get them both anyway because that's how your option parser works, but yeah, the second description there is superfluous.